docs/standaard/documenten/consultatie-v1_5_0.md

```diff
@@ -0,0 +1,223 @@
+
+__Merk op dat de áfwezigheid van persoonsgegevens niet kan worden gezien als vrijbrief voor openbare publicatie. Er kunnen immers andere beperkingsgronden (vertrouwelijk gedeelde bedrijfs- en fabricagegegevens, schending van belangen gediend met opsporing en vervolging van strafbare feiten, ...) of auteursrechtelijke beperkingen gelden.__
+
+## Wat vervalt (op termijn)?
+
+Belanghebbenden gaven aan bovenstaande wijziging bij voorkeur verwerkt te zien in een _minor_ release van de Documenten API. Dit heeft als postief gevolg dat de overgang naar nieuwe property's en waarden niet in één 'big bang' hoeft te gebeuren. Maar het betekent dat het 'oude' niet zonder meer uit de specificatie mag verdwijnen, maar dat sprake moet zijn van een overgangsperiode waarin oud en nieuw naast elkaar kunnen bestaan. Dit kan worden bereikt door middel van 'deprication'. Met name de property `status` vormt hierbij een uitdaging. Daarvoor geldt immers dat sommige toegestane waarden geheel nieuw zijn, sommige een (door introductie van nieuwe waarden) geactualiseerde betekenis hebben gekregen, terwijl één waarde `gearchiveerd` geheel vervalt. Om aan te geven dat we willen dat 'oude' waarden zo min mogelijk worden gebruikt, zagen we drie mogelijkheden:
+
+1. Het toekennen van het 'depricated'-label aan individuele enum-waarden. Dit is, volgens de OpenAPI-specificaties versie 3.0 die we voor deze standaarden hanteren, [niet mogelijk](https://stackoverflow.com/questions/63963150/with-swagger-2-0-or-even-3-0-is-it-possible-to-mark-an-enum-value-as-deprecat).
+2. Het opnemen van een geheel nieuwe property `statusv2` (of iets dergelijks), terwijl aan de bestaande property `status` het 'depricated'-label wordt toegekend. Dit leidt ertoe dat we ofwel tot in lengte van dagen met een lelijke, niet descriptieve propertynaam met daarin een versieaanduiding blijven zitten, ofwel later nog een wijziging moeten doorvoeren om de 'v2'-suffix weer te verwijderen. Om deze reden is hiervoor niet gekozen.
+3. Het toevoegen van een nieuwe enum met daarin nieuwe toegestane statuswaarden onder de naam `StatusEnumV2` en het toekennen van het 'depricated'-label aan de bestaande enum `StatusEnum`. Hoewel hierbij dezelfde suffix-problematiek speelt als bij 2. hierboven, hebben we geoordeeld dat dit in de naam van een enum minder erg is dan in de naam van een property. Voor deze oplossing is derhalve gekozen.
+
+In de volgende _major_ release van de Documenten API kan de enum `StatusEnum` geheel komen te vervallen.
+
+## Aandachtspunten bij deze consultatie
+
+### Interacties tussen API-consumers en -providers
+
+Depricaten van de statuswaarden zoals die versies van Documenten API > 1.5.0 werden gehanteerd heeft een aantal consequenties:
```

Comment:
Typo's:

- "die versies" -> "die in versies"
- "Documenten API > 1.5.0" -> "Documenten API < 1.5.0"